assets/src/components/v2/cr_departures.tsx

```diff
@@ -187,7 +168,7 @@ const CRDepartures: React.ComponentType<CRDeparturesProps> = (props) => {
             </div>
             <div className="departures-card__footer-ride-free">
               <div className="departures-card__footer-english ride-free">
-                Show your CharlieCard or CharlieTicket to ride at no charge
+                Show your Charlie Card or Charlie Ticket to ride free of charge
```

Comment:
Big merge conflict: `free of charge` is forbidden 🙄 the above line is the correct terminology.